datalad/distributed/tests/test_ora_http.py

```diff
@@ -115,7 +115,17 @@ def test_read_access(store_path, store_url, ds_path):
     create_store(io, store_path, '1')
     create_ds_in_store(io, store_path, ds.id, '2', '1')
     ds.repo.init_remote('ora-remote', options=init_opts)
-    ds.repo.fsck(remote='ora-remote', fast=True)
+    fsck_results = ds.repo.fsck(remote='ora-remote', fast=True)
+    # Note: Failures in the special remote will show up as a success=False
+    # result for fsck -> the call itself would not fail. Hence, assert
+    # nothing special to be in "note" or "error-messages".
+    for r in fsck_results:
+        if "note" in r:
+            # we could simply assert "note" to not be in r, but we want proper
+            # error reporting - content of note, not just its unexpected
+            # existence.
+            assert_equal(r["note"], "")
+        assert_equal(r["error-messages"], [])
```

Comment:
Why is it not checking for the `success=true` result property? This is the direct indicator, while `note` and `error-messages` are more like proxies.

Reply:
?

Reply:
I wanted the message to show up, when the assertion fails. But you're right - can be achieved differently.